src/main/java/org/embulk/util/aws/credentials/AwsCredentials.java

```diff
@@ -171,13 +200,40 @@ public void refresh() {
             };
         }
 
+        case "role":
```

Comment:
IIUC, AWS IAM has a different authentication method called `PassRole`, than `AssumeRole`. This `role` is for `AssumeRole`.

May we want to call it `assume_role` (or `assumeRole`?), not just `role`, so that users are not confused with `PassRole`?

Reply:
I have change `role` to `assume_role`

Reply:
It's okay to me